doc/sphinx/language/extensions/evars.rst

```diff
@@ -74,18 +74,51 @@ Expressions often contain redundant pieces of information. Subterms that can be
 automatically inferred by Coq can be replaced by the symbol ``_`` and Coq will
 guess the missing piece of information.
 
+Automatic resolution of existential variables
```

Comment:
We don't use title case for these headers right?

Reply:
We use title case.  This one shows up in the table of contents panel.

Reply:
I don't get your answer. Title case means:

>When using title case, all words are capitalized, except for minor words that are not the first or last word of the title.

I see no title using this style anywhere in the refman.

Reply:
My bad, not sure what I was thinking.